### DISCUSSION TITLE
Daily 16/06

### DISCUSSION THREAD

**Discussion by NataliaAlvarezIspc** [NataliaAlvarezIspc/proyecto-ispc-ecommerce#636] [Daily Sprint]
Con respecto al proyecto:

¿Qué hice ayer?
¿Qué haré hoy?
Impedimentos o dificultades

**Comment by NataliaAlvarezIspc** [comment]
Ayer estuve revisando la documentación e hice algunas modificaciones, faltaría ver como queda el diagrama relacional para terminar con esa parte. Por la tarde tuve clases de programación. También estuve viendo la funcionalidad de la página buscando errores, cree algunas tareas al respecto. 
Hoy quiero terminar con toda la parte de documentación para ya tenerlo subido. Por la noche nos juntaremos para hacer el demo.

**Comment by rpgrca** [comment]
Último día (espero)! Ayer arreglé varios bugs, hoy arreglé el del refresco del header al loguearse, estoy el mensaje al comprar en efectivo y voy a ver si puedo hacer lo de modificar usuarios durante el día. A la tarde tengo inglés y a la noche a las 20 nos juntamos para tratar de grabar hasta las 21 y si no podemos nos volvemos a juntar a las 22. Después probar durante el sábado para ver si no queda algo suelto y si cumplimos con todo lo que piden en los sprints y en todo caso si nos dan la devolución de los otros sprints terminar lo que haga falta hacer.

**Comment by BraianTroncoso** [comment]
Ayer intente solucionar unos bugs lo cual me trabé y los realizó Roberto y me compartió pantalla para aprender como él los resolvia, lo cual me vino barbaro. También tuve clases de Programación. Hoy a las 20hs nos vamos a reunir a grabar explicando el proyecto y el código.
Voy a probar la web para ver si encuentro algun bug y agregarlo como tarea para resolverlo.
